homeassistant/components/sensor/recorder.py

```diff
@@ -120,7 +120,7 @@
 WARN_UNSTABLE_UNIT = "sensor_warn_unstable_unit"
 
 
-def _get_entities(hass: HomeAssistant) -> list[tuple[str, str, str | None]]:
+def _get_entities(hass: HomeAssistant) -> list[tuple[str, str, str | None, str | None]]:
```

Comment:
a namedtuple would help making this code a bit more clear.

Reply:
Good idea for a follow-up 👍